NitroxClient/GameLogic/Rockets.cs

```diff
@@ -76,5 +83,54 @@ public void CallElevator(NitroxId id, RocketElevatorPanel panel, bool up)
                 Log.Error($"{nameof(Rockets.CallElevator)}: Can't find model for rocket with id {id}");
             }
         }
+
+        public void RequestRocketLaunch(Rocket rocket)
+        {
+            if (NitroxEntity.TryGetEntityFrom(rocket.gameObject, out NitroxEntity entity))
+            {
+                packetSender.Send(new RocketLaunch(entity.Id));
+            }
+            else
+            {
+                Log.Error($"{nameof(Rockets.RequestRocketLaunch)}: Can't find a NitroxEntity attached to the Rocket: {rocket.name}");
+            }
+        }
+
+        public void RocketLaunch(NitroxId rocketId)
+        {
+            // Avoid useless calculations
+            if (LaunchRocket.launchStarted)
+            {
+                return;
+            }
+
+            GameObject rocketObject = NitroxEntity.RequireObjectFrom(rocketId);
+            GameObject sphereCenter = rocketObject.FindChild("AtmosphereVolume");
+            LaunchRocket launchRocket = rocketObject.RequireComponentInChildren<LaunchRocket>(true);
+
+            // Only launch if you're in the rocket so
+            // verify if the distance to a centered point in the middle of the stage 3 of the rocket is inferior to 5.55 (pre-calculated radius)
+            if (Player.main.IsUnderwater() ||
+                Player.main.currentSub ||
+                NitroxVector3.Distance(Player.main.transform.position.ToDto(), sphereCenter.transform.position.ToDto()) > 5.55f)
+            {
+                return;
+            }
+
+            // When the server sends this to the client, he should execute the rocket launch
+            // Code extracted from LaunchRocket::OnHandClick
+            LaunchRocket.SetLaunchStarted();
+            PlayerTimeCapsule.main.Submit(null);
+            launchRocket.StartCoroutine(launchRocket.StartEndCinematic());
+            HandReticle.main.RequestCrosshairHide();
+
+            // We also need to hide the other players
+            foreach (RemotePlayer player in playerManager.GetAll())
+            {
+                player.PlayerModel.SetActive(false);
+            }
+
+            ErrorMessage.AddMessage("Thanks for using Nitrox!");
```

Comment:
This should be translatable imo.

Reply:
I think the language key should be "Nitrox_ThanksForPlaying" and not "Nitrox_ThankForPlaying", or am I wrong here?

Also it would be better that you add this key to en.json (and only that file).

Reply:
From the verb "to thank" was my intention

Reply:
Ah ok, yeah makes sense